app/serializers/note_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class NoteSerializer < ActiveModel::Serializer
+  attributes :id, :title, :body
```

Comment:
This will need adjusted to reflect the output requested (and that the test file expects)